packages/angular/cli/src/command-builder/utilities/schematic-engine-host.ts

```diff
@@ -12,6 +12,7 @@ import { readFileSync } from 'fs';
 import { parse as parseJson } from 'jsonc-parser';
 import { createRequire } from 'module';
 import { dirname, resolve } from 'path';
+import { TextEncoder } from 'util';
```

Comment:
I used an `import` for this, even though it's available globally, because for some reason the global variable wasn't in the typings.